### PR TITLE
Various minute improvements

### PR DESCRIPTION
**Pull request by lea-weissman**
As the title states.

### CODE REVIEW

**Comment by monoidic** [comment]
LGTM

File: services/flow2pwn.py
Comment (Diniboy1123) [inline services/flow2pwn.py:1]: LGTM, but next time don't forget to adjust the Copyright statement

**Review by monoidic — APPROVED** [review]

File: services/flow2pwn.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def flow2pwn(flow):
 
     for message in flow['flow']:
         data = base64.b64decode(message["b64"])
+        if finder := match(b'bd-executor: (.{50})', data):
```
Comment (rebane2001) [inline services/flow2pwn.py:52]: LGTM, +1 for the 50 character limit to prevent DoS

Comment (Speedy11CZ) [inline services/flow2pwn.py:52]: meme

**Review by DaClemens — CHANGES_REQUESTED** [review]
Thank you for your first contribution!

I do feel however that your patch is lacking a little bit of documentation. Coukd you explain what bug you're trying to fix, and how you observed this bug? We at the Tulip Development Sweatshop really take feedback seriously, so unfortunately we can't accept patches that don't have any feedback attached with them.

Also, if you have the time, please take a look at #17, it looks to me like these are probably related.

Ping me once you got that covered, and I'll gladly merge!

**Comment by RickdeJager** [comment]
Nice try ;)

**Pull request closed without merging**